cpp/velox/substrait/VeloxSubstraitSignature.cc

```diff
@@ -182,6 +183,43 @@ TypePtr VeloxSubstraitSignature::fromSubstraitSignature(const std::string& signa
     return ARRAY(elementType);
   }
 
+  if (startWith(signature, "map")) {
+    // Map type name is in the format of map<T1,T2>.
+    auto mapStart = signature.find_first_of('<');
```

Comment:
There are so much code which copied from `struct` type, can we extract them?

Reply:
ok

Reply:
fixed